build/cqs.tsv

```diff
@@ -73,10 +80,12 @@ log4j:log4j	1.2.17	compile
 net.iharder:base64	2.3.8	compile
 net.java.dev.jsr-275:jsr-275	1.0-beta-2	compile
 net.jpountz.lz4:lz4	1.2.0	compile
+net.jpountz.lz4:lz4	1.3.0	compile
```

Comment:
Awaiting PMC.

Where is net.jpountz.lz4 1.2.0 coming from?  We don't seem to have a CQ for that version?